### PR TITLE
Update information about imagepruner object

### PR DESCRIPTION
**Pull request by dmage**
4.4 has only keepYoungerThan that is an integer in nanoseconds.

https://bugzilla.redhat.com/show_bug.cgi?id=1840973

/assign @bmcelvee

### CODE REVIEW

File: modules/pruning-images.adoc [hunk 1]
```diff
@@ -25,20 +20,19 @@ When the Pruning Custom Resource is deleted, the pruning `CronJob` and its relat
 [source,yaml]
 ----
 spec:
-  schedule: "0 0 * * *" <1>
+  schedule: 0 0 * * * <1>
   suspend: false <2>
   keepTagRevisions: 3 <3>
-  keepYoungerThan: 60m <4>
+  keepYoungerThan: 3600000000000 <4>
```
Comment (bmcelvee) [inline modules/pruning-images.adoc:26]: Should this be 3600000000000s?

File: modules/pruning-images.adoc [hunk 2]
```diff
@@ -55,29 +49,28 @@ status:
...
     message: "Most recent image pruning job succeeded."
 ----
-<1> `schedule`: `CronJob` formatted schedule, defaults to daily at midnight for new clusters. This is a required field.
-<2> `suspend`: If set to `true`, the `CronJob` running pruning is suspended. This is an optional field, and it defaults to `false`.
-<3> `keepTagRevisions`: The number of revisions per tag to keep. This is an optional field, and it defaults to `3` if not set.
-<4> `keepYoungerThan`: Retain images younger than this duration. This is an optional field, and it defaults `60m` if not set.
+<1> `schedule`: `CronJob` formatted schedule. This is an optional field, default is daily at midnight.
+<2> `suspend`: If set to `true`, the `CronJob` running pruning is suspended. This is an optional field, default is `false`. The initial value on new clusters is `true`.
+<3> `keepTagRevisions`: The number of revisions per tag to keep. This is an optional field, default is `3`. The initial value is `3`.
+<4> `keepYoungerThan`: Retain images younger than this duration in nanoseconds. This is an optional field, default is `3600000000000` (60 minutes).
```
Comment (bmcelvee) [inline modules/pruning-images.adoc:55]: Should we specify seconds here too?

**Comment by bmcelvee** [comment]
Just a couple of questions about specifying seconds, otherwise LGTM! Thanks, @dmage!

@wzheng1 would you mind taking a look too, thanks!

File: modules/pruning-images.adoc (hunk 1 above)
Comment (wzheng1) [reply modules/pruning-images.adoc:26]: No, just an integer is fine.

**Comment by wzheng1** [comment]
The change looks good to me! Thanks! @bmcelvee There will be an update to 4.5 later, right?

File: modules/pruning-images.adoc (hunk 1 above)
Comment (dmage) [reply modules/pruning-images.adoc:26]: It should be an integer (so it cannot have suffixes), and it's in **nano**seconds.

**Comment by dmage** [comment]
Update to 4.5: #23859

File: modules/pruning-images.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/pruning-images.adoc:26]: Ah, thanks!

**Comment by bmcelvee** [comment]
Thank you, @dmage and @wzheng1! Going ahead with the merge as soon as the Travis check is complete and green.

**Pull request merged**